examples/chip-tool/commands/clusters/Commands.h

```diff
@@ -4978,8 +4970,9 @@ class GeneralCommissioningSetFabric : public ModelCommand
 
         chip::Controller::GeneralCommissioningCluster cluster;
         cluster.Associate(device, endpointId);
-        return cluster.SetFabric(onSuccessCallback->Cancel(), onFailureCallback->Cancel(), reinterpret_cast<char *>(mFabricId),
-                                 reinterpret_cast<char *>(mFabricSecret), mBreadcrumb, mTimeoutMs);
+        return cluster.SetFabric(onSuccessCallback->Cancel(), onFailureCallback->Cancel(), reinterpret_cast<uint8_t *>(mFabricId),
+                                 static_cast<uint32_t>(strlen(mFabricId)), reinterpret_cast<uint8_t *>(mFabricSecret),
```

Comment:
This doesn't compile, afaict.

Also, why would we not just store the BytesData type in `mFabricId`?

Reply:
Fixed compiling, the mFabricId is a string read from command line in chip-tool, we might add a parser to convert it to a byte array later.